stage1_fly/aci/aci-manifest.in

```diff
@@ -0,0 +1,33 @@
+{
+    "acKind": "ImageManifest",
+    "acVersion": "0.7.3",
+    "name": "coreos.com/rkt/stage1-fly",
+    "labels": [
+        {
+            "name": "version",
+            "value": "@RKT_STAGE1_VERSION@"
+        },
+        {
+            "name": "arch",
+            "value": "amd64"
+        },
+        {
+            "name": "os",
+            "value": "linux"
+        }
+    ],
+    "annotations": [
+        {
+            "name": "coreos.com/rkt/stage1/run",
+            "value": "/run"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/enter",
+            "value": "/enter"
```

Comment:
The doc [stage1-implementors-guide.md](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md) should say what's happening when the dictionary entries are missing:

- run: error out
- gc: silently skip the execution of stage1's GC
- enter: error out with "not implemented"